modules/airgap/instance/instance_server.tf

```diff
@@ -0,0 +1,144 @@
+data "template_file" "is_airgap" {
+    template = (var.enable_public_ip == false && var.enable_ipv6 == false) ? true : false
+}
+
+data "template_file" "is_ipv6only" {
+    template = (var.enable_public_ip == false && var.enable_ipv6 == true) ? true : false
+}
+
+resource "aws_instance" "master" {
+
+  depends_on = [ null_resource.prepare_bastion ]
+
+  ami                         = var.aws_ami
+  instance_type               = var.ec2_instance_class  
+  associate_public_ip_address = false
+  ipv6_address_count          = var.enable_ipv6 ? 1 : 0
+  count                       = var.no_of_server_nodes
+  
+  root_block_device {
+    volume_size          = var.volume_size
+    volume_type          = "standard"
+  }
+  subnet_id              = var.subnets
+  availability_zone      = var.availability_zone
+  vpc_security_group_ids = [var.sg_id]
+  key_name               = var.key_name
+  tags = {
+    Name                 = "${var.resource_name}-server${count.index + 1}"
+  }
+  user_data              = <<-EOF
+                             #!/bin/bash
+                             sudo mkdir -p /etc/rancher/${var.product}
+                           EOF  
+
+}
+
+resource "aws_instance" "worker" {
```

Comment:
here on mine this is broken, on your end is good ? 
may tf version, idk

Reply:
Yeah its good on my end, not sure what is broken...